cmd/ssl-selfsign-handshake-check/ssl-selfsign-handshake-check.yaml

```diff
@@ -15,7 +15,7 @@ spec:
           # If not using default SSL port of 443, port env variable must be updated  
           - name: PORT
             value: "443"
-        image: kuberhealthy/ssl-selfsign-handshake:v1.2.0
```

Comment:
If you're changing image names, be sure that the github action (under `.github/workflows` at the root) has the right info in it.  I would also modify the `Makefile` for the check so that it builds with the right image name and tag.

Reply:
Github actions under workflows should all match current image names of `_check-name_-check`.

All Makefiles match current check names, and are set to current versions, 1.2.0.